app/src/main/java/org/mifos/mobile/ui/beneficiary_detail/BeneficiaryDetailContent.kt

```diff
@@ -0,0 +1,95 @@
+package org.mifos.mobile.ui.beneficiary_detail
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.MifosTitleDescSingleLineEqual
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.beneficiary.Beneficiary
+
+@Composable
+fun BeneficiaryDetailContent(
+    beneficiary: Beneficiary?
```

Comment:
beneficiary should not be null in here, 
check in the caller itself, if its null, show error ui state